trillian_log_api.proto

```diff
@@ -216,6 +235,15 @@ service TrillianLog {
       };
     }
 
+    // AddSequencedLeaf adds a single leaf with assigned sequence number to the
+    // tree's storage, to be integrated later to the tree.
+    rpc AddSequencedLeaf (AddSequencedLeafRequest) returns (AddSequencedLeafResponse) {
+      option (google.api.http) = {
+        post: "/v1beta1/logs/{log_id}/leaves:sequenced"
```

Comment:
consider
`post: "/v1beta1/logs/{log_id}/leaves/{leaf_id}`

Reply:
I assume `leaf_index` to be consistent with `LogLeaf.leaf_index`.

It requires me to add a `leaf_index` field to `AddSequencedLeafRequest`. Is it possible to map this directly to `AddSequencedLeafRequest.leaf.leaf_index`? I tried putting `{leaf.leaf_index}` here, doesn't work.